init/lib/lib_test.go

```diff
@@ -18,22 +18,33 @@ import (
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strconv"
 	"strings"
 	"testing"
 
 	"github.com/stretchr/testify/require"
+	"gopkg.in/yaml.v2"
+
+	"github.com/palantir/go-java-launcher/launchlib"
 )
 
-var files = []string{LauncherStaticFile, LauncherCustomFile, OutputFile, Pidfile}
+var files = []string{launchlib.LauncherStaticFile, launchlib.LauncherCustomFile, launchlib.OutputFileFormat, pidfile}
```

Comment:
This is why I `defer teardown()` so that if something panics then this stuff gets cleaned up. If one terminates the program in such a way that deferred functions are not run, it's reasonable to expect that certain things the program was doing might not be cleaned up.

Reply:
I do not think that is reasonable, particularly given one way of it happening is just Crtl+C'ing the test run.

Reply:
Ok, installed signal handlers so that things will get cleaned up on ctrl+c (among other signals).

Reply:
thanks